donate/templates/pages/core/ways_to_give_page.html

```diff
@@ -0,0 +1,94 @@
+{% extends "pages/base_page.html" %}
```

Comment:
Where'd the "Mozilla" nav bar at the top of the rendered page go?

Reply:
Ah didn't know we wanted it since we are imitating https://donate.mozilla.org/en-US/ways-to-give and that page doesn't have a nav. 

I'll add nav back here.